src/web/src/app/space/list/component.html

```diff
@@ -0,0 +1,116 @@
+<h1 i18n="@@SPACES.LIST.HEADING">Пространства</h1>
+<ng-container *ngIf="invitationsTotal > 0">
+	<hr class="width-and-clear">
+	<h2 i18n="@@SPACES.LIST.INVITES.HEADING">Приглашения</h2>
+	<div #scrollToInvitationForm class="spaces-invitations-wrap">
+		<!-- form for inviting user -->
+		<div class="float-row-with-padding">
+			<form [formGroup]="invitationForm" [hidden]="isInvitationFormHidden" (ngSubmit)="onSubmitInvitation(invitationForm)">
```

Comment:
hidden тут выглядит как лишний трюк. Давай лучше совсем спрячем этот div.
```suggestion
		<div class="float-row-with-padding" *ngIf="isInvitationForm">
			<form [formGroup]="invitationForm" (ngSubmit)="onSubmitInvitation()">
```

Reply:
Получается, тут нужно при нажатии на кнопку раскрытия формы создавать форму, чтобы ngIf сработал? А при закрытии формы нужно переменной формы присваивать null? Ок, на завтра запланировал.

Reply:
Готово.